<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Header</class>
 <widget class="QWidget" name="Header">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}

QLabel
{
	color: white;
	font: 20px, &quot;微软雅黑&quot;;
}</string>
  </property>
  <widget class="QPushButton" name="btnHelp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 40px,  &quot;微软雅黑&quot;;;
	color: rgb(0, 255, 0);
    background-color: rgb(200, 200, 200);
}</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPageTitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: white;
	font: 28px, &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>车辆状态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRunningMode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ATP模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTractionPercent">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>牵引－0％</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblDirection">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>向前 ▶</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNextStation">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>下一站：－</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCurrentStation">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前站：－</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>98</y>
     <width>1024</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white</string>
   </property>
   <property name="text">
    <string>ATP模式</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>749</x>
     <y>30</y>
     <width>271</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblDate">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>2016-03-03</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTime">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>15:27:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblVoltage">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0V</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblSpeed">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0 km/h</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblNoCommunication">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: red;
	font: 28px, &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>通讯中断</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
